src/meta/src/hummock/manager/mod.rs

```diff
@@ -1290,6 +1303,11 @@ where
             );
         }
 
+        if task_status == TaskStatus::Success {
```

Comment:
minor: can we move this update in is_success branch

Reply:
is_success branch holds versioning lock, meanwhile `try_update_write_limits` need require that lock too.
So I'd like to keep this update as it is, instead of providing another `try_update_write_limits` with versioning lock as parameter.